src/apps/intel/loadgen.lua

```diff
@@ -14,9 +14,10 @@ local can_transmit, transmit
 
 LoadGen = {}
 
-function LoadGen:new (pciaddress)
-   local o = { pciaddress = pciaddress,
-               dev = intel10g.new_sf({pciaddr=pciaddress}) }
+function LoadGen:new (conf)
+   local o = { pciaddr= conf.pciaddr,
+               dev = intel10g.new_sf({pciaddr=conf.pciaddr}),
+               report_rx = conf.report_rx, }
```

Comment:
What exactly do the changes to `LoadGen` do? Does the `LoadGen` documentation need an update to reflect `conf.report_rx`?